src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritManagement.java

```diff
@@ -170,10 +183,10 @@ public AutoCompletionCandidates doAutoCompleteCopyNewItemFrom(@QueryParameter fi
     }
 
     /**
-    * Gets the list of GerritServer.
-    *
-    * @return the list of GerritServer.
-    */
+     * Gets the list of GerritServer.
```

Comment:
Darn! I never asked for these to be changed. must have been a slip when I optimized the imports.